### DISCUSSION TITLE
Building vs. Using

### DISCUSSION THREAD

**Discussion by metabsd** [vmware/packer-examples-for-vsphere#80] [Help] [answered]
Hi,
I understand that to build the VM the best method is to use DHCP. 
However I would like to know if after building the VM why it still tries to use DHCP even if I use a vm customization by specifying a static address. 
I would like to be able to build with dhcp but consume with static addresses. 

Thanks for your help!

**Comment by metabsd** [comment]
Do you know if this is normal that the kernel keep all the parameter ?

![image](https://user-images.githubusercontent.com/7953470/135645155-d77eafcf-bdca-4c04-9bf7-31c0b0a13810.png)

**Comment by tenthirtyam** [comment]
It might be related to cloud-init.

**Reply by metabsd** [reply]
Not sure if related but I found this.
https://discuss.hashicorp.com/t/error-in-packer-vsphere-iso-for-ubuntu-20-04-3/29869

**Reply by tenthirtyam** [reply]
Hey! Thanks for sharing. I'm already taking a looking into this for an upcoming update I'm working on that will use 20.04.3. I was seeing something similar with 2@.04.2.

Ryan

**Comment by tenthirtyam** [comment, marked as answer]
If you try today's `main`, the Ubuntu 20.04 issues should be cleared up from some fixes in #93.

I made some changes to the instatll `boot_command` and `user-data` to clear up some items. 

Ryan

**Reply by metabsd** [reply]
I confirm that the latest version work fine for me :) 

Thx a lot!!

**Reply by tenthirtyam** [reply]
Sweet!
